test/Microsoft.Framework.Runtime.Tests/project.json

```diff
@@ -2,6 +2,7 @@
     "dependencies": {
         "Microsoft.Framework.Runtime": "1.0.0-*",
         "Microsoft.Framework.Runtime.Interfaces": "1.0.0-*",
+        "Newtonsoft.Json": "6.0.6",
```

Comment:
Why did you have to add this? Don't get it transitively  via Microsoft.Framework.Runtime?